### DISCUSSION TITLE
boot loop Android 10. LSPosed version 1.6.3

### DISCUSSION THREAD

**Discussion by StinkyMint** [LSPosed/LSPosed#1406] [Q&A] [answered]
after install lsposed 1.6.3 to Android 10 phone don't load

After downgrade to lsposed 1.5.3 (5984) all work correctly

**Comment by yujincheng08** [comment, marked as answer]
Fixed here: https://github.com/LSPosed/LSPosed/commit/8ebdcd55b4981d4a36a736e69b9d357559ab1a02

**Comment by StinkyMint** [comment]
Confirmed.
Update work nicely after your fix.
Thanks.
